src/main/java/ay2021s1_cs2103_w16_3/finesse/model/budget/MonthlyBudget.java

```diff
@@ -7,9 +7,14 @@
  */
 public class MonthlyBudget {
     private MonthlyExpenseLimit monthlyExpenseLimit;
+    private MonthlySavingsGoal monthlySavingsGoal;
 
+    /**
+     * Creates a {@code MonthlyBudget}.
```

Comment:
```suggestion
     * Creates a {@code MonthlyBudget} with an expense limit and savings goal of $0.
```

Comment is not descriptive.